testsuite/utils/src/main/resources/META-INF/keycloak-server.json

```diff
@@ -33,7 +33,12 @@
     },
 
     "globalLock": {
-        "provider": "${keycloak.globalLock.provider:dblock}"
+        "provider": "${keycloak.globalLock.provider:dblock}",
+        "map": {
+            "storage": {
+                "provider": "${keycloak.lock.map.storage.provider,keycloak.mapStorage.provider.default:concurrenthashmap}"
+            }
+        }
```

Comment:
Good point, the next commit will add it also to the pom.xml for the integration tests so that the integration tests when running with JPA use it as well.